src/components/common/community.tsx

```diff
@@ -126,20 +136,17 @@ const Community: React.FC<CommunityProps> = (props) => {
   return (
     <Wave
       background="orange"
-      direction={props.direction}
+      direction={direction}
       wave="blue"
       spacing="lg"
       className="flex items-center justify-center"
     >
       {!submitted && (
-        <div className="max-w-xl">
+        <div>
```

Comment:
Since there is no class on this anymore we should switch this to a react fragment:

Instead of wrapping this in an empty div such as `<div>Content</div>` instead just do `<>Content</>`

Reply:
@Stormtv div is needed or else it goes to inline and messes up the whole thing